charts/kubefed/values.yaml

```diff
@@ -6,19 +6,36 @@
 ##
 controllermanager:
   enabled: true
-  replicaCount: 2
-  repository: quay.io/kubernetes-multicluster
-  image: kubefed
-  tag: canary
-  imagePullPolicy: IfNotPresent
-  logLevel: 2
-  resources:
-    limits:
-      cpu: 500m
-      memory: 512Mi
-    requests:
-      cpu: 100m
-      memory: 64Mi
+  manager:
+    replicaCount: 2
+    repository: quay.io/kubernetes-multicluster
+    image: kubefed
+    tag: canary
+    imagePullPolicy: IfNotPresent
+    logLevel: 2
+    env: {}
+    resources:
+      limits:
+        cpu: 500m
+        memory: 512Mi
+      requests:
+        cpu: 100m
+        memory: 64Mi
+  webhook:
```

Comment:
Why we put `webhook`'s properties under `controllermanager`?

Reply:
They are part of the chart components named controller manager.

Reply:
The webhook can actually be running external to the application, however this was for the sake of simplicity in the first place, probably not to define many new additional settings.

Reply:
I didn't want to reorganize the whole chart before we get this potential PR with Helmv3 support. But I could do it if you want it.

Reply:
I did not imply to move it out. I am ok until there is a need for it. :) (YAGNI)